src/Features/Core/Portable/Navigation/NavigableItemFactory.cs

```diff
@@ -79,51 +77,5 @@ private static IEnumerable<Location> GetPreferredSourceLocations(ISymbol symbol)
                 ? navigableItems.Where(n => !n.Document.IsGeneratedCode(cancellationToken))
                 : navigableItems.Where(n => n.Document.IsGeneratedCode(cancellationToken));
         }
-
-        public static ImmutableArray<TaggedText> GetSymbolDisplayTaggedParts(Project project, ISymbol symbol)
```

Comment:
not needed at all anymore.  We don't go back to symbols to compute anything for NavigateTo items.